Diagnostics/Get-NetView.PS1

```diff
@@ -836,7 +836,7 @@ function NativeNicDetail {
 function ChelsioDetail {
     [CmdletBinding()]
     Param(
-        [parameter(Mandatory=$true)] [String] $IfIndex,
+        [parameter(Mandatory=$true)] [CimInstance] $Nic, # Get-NetAdapter output
```

Comment:
Use -NicName here for consistency with other CMDLETs input parameters.  Similar to prior comment.